website/templates/emails/no_addon.html.mako

```diff
@@ -0,0 +1,25 @@
+## -*- coding: utf-8 -*-
+<%inherit file="notify_base.mako"/>
+<%def name="content()">
+    <div style="margin: 40px;">
+    <br>
+        Hello ${fullname},
+        <br>
```

Comment:
Can you add a space after the greeting?
![screen shot 2015-09-21 at 12 57 12 pm](https://cloud.githubusercontent.com/assets/6414394/9999381/1c76bd54-6064-11e5-9b1d-10237ff94b3e.png)